### PR TITLE
Move the source code from actions-gh-release repository

### PR DESCRIPTION
**Pull request by knanao**
**What this PR does / why we need it**:
#3213 

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: dockers/actions-plan-preview/planpreview_test.go [hunk 1]
```diff
@@ -15,13 +15,19 @@
...
 import (
+	"os"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 func TestMakeCommentBody(t *testing.T) {
+	// NOTE: Skip this temporarily since this test result is diiffrent on CI and Local env.
```
Comment (knanao) [inline dockers/actions-plan-preview/planpreview_test.go:26]: https://github.com/pipe-cd/pipecd/runs/5827674246?check_suite_focus=true

File: dockers/actions-gh-release/DOCKER_BUILD [hunk 2]
```diff
@@ -0,0 +1,2 @@
+version: 2.3.3
```
Comment (nghialv) [inline dockers/actions-gh-release/DOCKER_BUILD:1]: How about bumping this to a new version. `2.3.4`

File: dockers/actions-gh-release/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
+FROM golang:1.16.5-alpine3.14
```
Comment (nghialv) [inline dockers/actions-gh-release/Dockerfile:1]: Let's use the same version used in the CI test. `1.17.6`

File: .github/workflows/dockers_test.yaml [hunk 4]
```diff
@@ -24,3 +24,7 @@ jobs:
       - name: Run unit tests for actions-plan-preview
         working-directory: dockers/actions-plan-preview
         run: go test ./...
+
+      - name: Run unit tests for actions-gh-release
+        working-directory: dockers/actions-gh-release
+        run: go test ./...
```
Comment (nghialv) [inline .github/workflows/dockers_test.yaml:30]: Could you fix the rename of this action from `name: Dockers Test` to `name: Test`

Comment (knanao) [reply .github/workflows/dockers_test.yaml:30]: That make sense!
and what do you think about also renaming of that job name from `Unit Tests` to `Docker Unit Tests`

Comment (nghialv) [reply .github/workflows/dockers_test.yaml:30]: @knanao You are right! Let's do it.

**Comment by nghialv** [comment]
Great job!

/lgtm

**Comment by khanhtc1202** [comment]
here you go
/approve

**Pull request merged**